article-analyzer/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.1.4'
+	id 'io.spring.dependency-management' version '1.1.3'
+}
+
+group = 'co.inventorsoft.academy'
+version = '0.0.1-SNAPSHOT'
+
+java {
+	sourceCompatibility = '17'
+}
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
all we need here are spring-boot-starter, gson and lombock
postgres, jpa, h2 - no
it is nice that you know these, but we will use these things later in the appropriate lectures.
here we only focused on Spring basic features, no fancy stuff